packages/react-native-session-replay/android/src/main/kotlin/com/datadog/reactnative/sessionreplay/ReactViewGroupMapper.kt

```diff
@@ -0,0 +1,105 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.reactnative.sessionreplay
+
+import android.annotation.SuppressLint
+import com.datadog.android.sessionreplay.internal.AsyncJobStatusCallback
+import com.datadog.android.sessionreplay.internal.recorder.MappingContext
+import com.datadog.android.sessionreplay.internal.recorder.mapper.BaseWireframeMapper
+import com.datadog.android.sessionreplay.internal.recorder.mapper.TraverseAllChildrenMapper
+import com.datadog.android.sessionreplay.model.MobileSegment
+import com.facebook.react.uimanager.Spacing
+import com.facebook.react.views.view.ReactViewBackgroundDrawable
+import com.facebook.react.views.view.ReactViewGroup
+
+internal class ReactViewGroupMapper :
+    BaseWireframeMapper<ReactViewGroup, MobileSegment.Wireframe>(),
+    TraverseAllChildrenMapper<ReactViewGroup, MobileSegment.Wireframe> {
+
+    @SuppressLint("VisibleForTests")
+    override fun map(
+        view: ReactViewGroup,
+        mappingContext: MappingContext,
+        asyncJobStatusCallback: AsyncJobStatusCallback
+    ): List<MobileSegment.Wireframe> {
+        val viewGlobalBounds = resolveViewGlobalBounds(
+            view,
+            mappingContext.systemInformation.screenDensity
+        )
+
+        val pixelDensity = mappingContext.systemInformation.screenDensity
+        val backgroundDrawable = view.background as? ReactViewBackgroundDrawable
+
+        val backgroundColor = backgroundDrawable?.color ?: 0
+        val viewAlpha = resolveAlphaValue(view.alpha, backgroundColor)
+
+        if (viewAlpha == 0) {
+            return emptyList()
```

Comment:
I see 2 corner cases that I'd like to check with this condition:
1. If there is another element inside the view that does not have an alpha of `0`, is it going to be rendered on the screen? if so, is it going to be in the replay as well?
2. If a View has an alpha of `0`, I think it still takes space on the screen? does it on the player as well? (I also think so since I think we use coordinates to place elements)

Reply:
With the current changes, 1 is handled and if the parent has an opacity of 0 then the children are not rendered. Regarding the second, we specify wireframes by absolute coordinates so I don't think it will be an issue. Let's discuss a test for this